src/shared/components/deletion-confirmation/DeletionConfirmation.test.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import DeletionConfirmation, {
+  DeletionConfirmationProps
+} from './DeletionConfirmation';
+
+const props: DeletionConfirmationProps = {
+  confirmText: 'Remove',
```

Comment:
Component and props talk about Delete and why the text says Remove?

Reply:
Its just a test and we are checking if we can change the text to anything else (can be delete or it can be any other text) as this is part of the component props.
The text and the props dont mention delete.

Reply:
Ah yes, good point!